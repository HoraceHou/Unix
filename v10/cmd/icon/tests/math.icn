procedure main()
   write(exp(2))
   write(log(2))
   write(log10(2))
   write(sqrt(2))
end
