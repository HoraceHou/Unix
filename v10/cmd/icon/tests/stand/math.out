7.3890561
0.69314718
0.30103
1.4142136
